classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_reader :students
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] You have set up the classroom for student. Now you should add the student to the classroom. To achieve this, your code should look like this:
```ruby
def add_student(student)
    @students << student
    student.classroom = self
end
```

Reply:
Hello @oloomoses, thanks for your review! I do push the student to the classroom's students array: If you take a look at the student's classroom setter (inside Student class), you'll find that I do it right there. Mind the screenshot:

![image](https://user-images.githubusercontent.com/70329406/146620595-81a29194-8cc6-4095-955a-c74d0cf58e3b.png)

As you can see, I add it on line 21. You can test it out and you'll find it works. I did run several tests and it never failed. Please let me know if it works for you.